src/index.js

```diff
@@ -0,0 +1 @@
+require('./server').listen();
```

Comment:
I meant, let's inline the `server.js` file into here, so we only have the `index.js` file. I don't see the need for this one liner. Does it just make testing easier?

Reply:
Ok, more things were added to this file, with the re-addition of starting service containers when container starts up.